### PR TITLE
feat(search-modal/zero-state): init UI recommendation and facilities

### PR DESCRIPTION
**Pull request by anggidast**
- [x] animal recommendation
- [x] public facilities
- [x] override light mode
- [x] fix keyboard directly appear
- [x] change current location point image
- [x] button start journey on overview for facilities
- [x] add icon app

### CODE REVIEW

**Comment by anggidast** [comment]
ready to review

File: Jeera/Views/Controllers/SearchViewController.swift [hunk 1]
```diff
@@ -10,29 +10,112 @@ import CoreLocation
...
+        searchResultTableView.backgroundColor = .white
+    }
+    
+    func mappingAnimalsRecommedation() {
+        for animalRecoms in ["Flamingo Eropa", "Sendok Raja", "Beruang Madu"] {
+            let animalsResult = animalsData.filter({ (animal: AllData) -> Bool in
+                let idNameMatch = animal.idName.range(of: animalRecoms, options: NSString.CompareOptions.caseInsensitive)
+                return idNameMatch != nil
+            })
+                animalsRecommendation.append(animalsResult.first!)
```
Comment (wildanbudi) [inline Jeera/Views/Controllers/SearchViewController.swift:101]: getting error on force unwrapped with nil value

Comment (wildanbudi) [inline Jeera/Views/Controllers/SearchViewController.swift:101]: getting error on force unwrapped with nil value

Comment (wildanbudi) [reply Jeera/Views/Controllers/SearchViewController.swift:101]: need loading screen

**Review by wildanbudi — APPROVED** [review]
lgtm

**Pull request merged**